server/views/formPages/openConditions/provisionalCategory.html

```diff
@@ -25,14 +25,23 @@
 
 <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
 
+{% set isFemale = user.activeCaseLoad.female %}
+
 <div class="govuk-warning-text forms-info-text warning-overrides">
-  <span class="govuk-warning-text__icon" aria-hidden="true">{{ cat }}</span>
+  {% if (not isFemale and cat == 'D') or isFemale %}
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  {% else %}
+    <span class="govuk-warning-text__icon" aria-hidden="true">{{ cat }}</span>
+  {% endif %}
   <strong class="govuk-warning-text__text">
     <span class="govuk-warning-text__assistive">Warning</span>
-    Based on the information provided, the provisional category is {{ catDisplay(cat) }}
+    {% if cat == 'D' %}
+      The provisional category is open
+    {% else %}
+      The provisional category is {{ catDisplay(cat) }}
```

Comment:
I think this should be `The provisional category is closed`

Reply:
I have checked this with Anthony:

For mens category D and womens category T the warning is: "! The provisional category is open"
For mens category B the warning is: "B The provisional category is category B
For mens category C the warning is: "C The provisional is category C"
For womens category R the warning is: "! The provisional category is closed"